README.md

```
@@ -11,18 +11,16 @@ This repo contains the **nano** **f**irmware **f**lasher tool.
 It's a [.NET Core Tool](https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools) that allows flashing a .NET **nanoFramework** target with a firmware image (nanoBooter and nanoCLR), the application deployment (all assemblies required to run a .NET application) and restore previously saved deployment images.
 Is part of .NET **nanoFramework** toolbox, along with other various tools that are required in .NET **nanoFramework** development, usage or repository management.
 
-It makes use of several 3rd party tools:
+It makes use of a couple of 3rd party tools for some platforms:
 
-- STM32 Cube Programmer
-   You can find the source, licensing information and documentation [here](https://www.st.com/en/development-tools/stm32cubeprog.html).
-- Texas Instruments Uniflash
+- Texas Instruments Uniflash (TI CC13x2/CC26x2)
    You can find the Uniflash tool and licensing information [here](http://www.ti.com/tool/download/UNIFLASH).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)